src/main/resources/db/migration/V003__Spring_session_schema.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE SPRING_SESSION (
+	SESSION_ID CHAR(36) NOT NULL,
+	CREATION_TIME BIGINT NOT NULL,
+	LAST_ACCESS_TIME BIGINT NOT NULL,
+	MAX_INACTIVE_INTERVAL INT NOT NULL,
+	PRINCIPAL_NAME VARCHAR(100),
```

Comment:
Use spaces, not tabs.
Also `cmd+alt+l` to format.

Reply:
flipping java people.